src/Raku/ast/signature.rakumod

```diff
@@ -94,7 +94,7 @@ class RakuAST::Signature
                         self.add-sorry: $resolver.build-exception: 'X::Parameter::WrongOrder', misplaced => 'required', after => 'named', parameter => $_.target.name;
                     }
 
-                } elsif $kind := 2 {           # optional
+                } elsif $kind =:= 2 {          # optional
```

Comment:
OOC, what's wrong with `==`  ??

Reply:
That's also something I wonder. I think == would be quite enough here. It is about numeric comparison after all.